src/Modules/Dashboard/Dashboard.tsx

```diff
@@ -56,7 +52,7 @@ export default function Dashboard() {
     Gettasks()
     Getmanagertask()
```

Comment:
We shouldn't capitalize the first letter of a function.

Reply:
done